sufia-models/app/models/sufia/works/curation_concern/with_editors.rb

```diff
@@ -0,0 +1,45 @@
+module Sufia::Works
```

Comment:
I don't think hydra-access-controls has a add_editor method.  We still may need the WithEditors mixin. @jcoyne ?